connection/src/main/java/com/ericsson/bss/cassandra/ecchronos/connection/DataCenterAwarePolicy.java

```diff
@@ -89,7 +89,8 @@ public void init(Map<UUID, Node> nodes, DistanceReporter distanceReporter)
         if (!notInLocalDC.isEmpty())
         {
             String nonLocalHosts = Joiner.on(",").join(notInLocalDC);
-            LOG.warn("Some contact points don't match local data center. Local DC = {}. Non-conforming contact points: {}", getLocalDatacenter(),
+            LOG.warn("Some contact points ({}) don't match local data center ({})",
+                    getLocalDatacenter(),
```

Comment:
Switch places for these 2 since you switched in the log msg.

Reply:
Done